backend/decompme/settings.py

```diff
@@ -49,8 +54,10 @@
     'django.contrib.sessions.middleware.SessionMiddleware',
     'corsheaders.middleware.CorsMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    #'django.middleware.csrf.CsrfViewMiddleware',
+    'coreapp.middleware.disable_csrf',
```

Comment:
I'd still be hesitant about disabling CSRF - but I don't feel strongly enough to hold up this review over it. (My web security knowledge is a few years out-of-date, so I'm leaning more conservative re: security)